autogen/safer-cluster/main.tf.tmpl

```diff
@@ -151,4 +151,6 @@ module "gke" {
   authenticator_security_group = var.authenticator_security_group
 
   enable_shielded_nodes = var.enable_shielded_nodes
+
+  skip_provisioners = var.skip_provisioners
```

Comment:
We need the ability to disable the provisioners if we're running in Terraform Cloud/Terraform Enterprise as `jq` and `gcloud` cannot be easily installed.